var/spack/repos/builtin/packages/turbovnc/package.py

```diff
@@ -0,0 +1,95 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Turbovnc(Package):
+    """TurboVNC is a derivative of VNC (Virtual Network Computing) 
+ that is tuned to provide peak performance for 3D and video workloads.
+ TurboVNC was originally a fork of TightVNC 1.3.x, on the surface, 
+ the X server and Windows viewer still behave similarly to their parents."""
+    # FIXME: add a proper url for your package's homepage here.
+    homepage = "http://www.turbovnc.org/"
+    url      = "http://downloads.sourceforge.net/project/turbovnc/2.0.1/turbovnc-2.0.1.tar.gz"
+
+
+    version('2.0.1', 'a279fdb9ac86a1ebe82f85ab68353dcc')
+    version('2.0.91', '1e203cc533103cf7bd1186c7fd620186',
+            url      = "http://downloads.sourceforge.net/project/turbovnc/2.0.91%20%282.1beta2%29/turbovnc-2.0.91.tar.gz"            
+    )
```

Comment:
No problem with that, and you can maintain the beta version in your fork.  For the PR into main Spack, I think the Beta version should be removed.  (I routinely maintain a Spack fork with things I need, but that should not go into the main Spack).